src/context/dancer/api-dao.ts

```diff
@@ -6,9 +6,11 @@ import { DancersDao } from 'context/dancer/types';
 const dancersApiDao = ({
   getIdTokenFunc,
   baseApiUrl,
+  awsUrl,
```

Comment:
Wherever we mention `awsUrl`, lets instead use `assetsUrl`. It isn't specific to aws, merely a location of assets.